streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryWindowStoreTest.java

```diff
@@ -98,7 +106,20 @@
     };
 
     private final File baseDir = TestUtils.tempDirectory("test");
-    private final InternalMockProcessorContext context = new InternalMockProcessorContext(baseDir, Serdes.ByteArray(), Serdes.ByteArray(), recordCollector, cache);
+    private final Properties props = StreamsTestUtils.getStreamsConfig();
+    private final StreamsConfig config = new StreamsConfig(props);
+    private final StreamsMetricsImpl streamsMetrics = new StreamsMetricsImpl(
+        new Metrics(new MetricConfig().recordLevel(Sensor.RecordingLevel.forName(config.getString(StreamsConfig.METRICS_RECORDING_LEVEL_CONFIG))))
```

Comment:
We need to enable DEBUG reporting level to test some lower-level sensors now. Ditto elsewhere.